src/main/java/com/coniverse/dangjang/domain/notification/entity/UserFcmToken.java

```diff
@@ -1,43 +1,65 @@
 package com.coniverse.dangjang.domain.notification.entity;
 
-import java.time.LocalDate;
-
 import com.coniverse.dangjang.domain.user.entity.User;
+import com.coniverse.dangjang.global.support.BaseEntity;
 
+import jakarta.persistence.EmbeddedId;
 import jakarta.persistence.Entity;
 import jakarta.persistence.FetchType;
-import jakarta.persistence.Id;
 import jakarta.persistence.JoinColumn;
 import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MapsId;
 import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 /**
  * UserFcmToken Entity
  * <p>
- * 사용자 모바일 기기의 fcmToken을 저장한다
+ * 사용자 모바일 기기의 fcmToken 을 저장한다
  *
  * @author EVE
  * @since 1.1.0
  */
 @Getter
 @Entity
 @NoArgsConstructor(access = lombok.AccessLevel.PROTECTED)
-public class UserFcmToken {
-	@Id
-	private String fcmToken;
+public class UserFcmToken extends BaseEntity {
+	@EmbeddedId
+	private FcmId fcmId;
```

Comment:
P3) fcmId를 getter로 노출시키지 않으려면 처리가 필요할 것 같아요 
```suggestion
	@Getter(AccessLevel.PRIVATE)
	private FcmId fcmId;
```

Reply:
그렇네요 ! 수정하겠습니다 :)